search-engines.lisp

```diff
@@ -1074,6 +1074,130 @@ REQUEST-ARGS is a list of args to pass to request function."
      :completion-function (make-pkgs-completion)
      :documentation "UNIX/GNU/Linux package search."))
 
+(define-search-engine peertube
+    (:shortcut "pt"
+     :fallback-url (quri:uri "search.joinpeertube.org")
+     :base-search-url "https:///search.joinpeertube.org/search?search=~a"
+     :documentation "Peertube `nx:search-engine' via its Sepia Search global search engine.")
```

Comment:
There's no `nx' nickname, isn't there? :p

Reply:
And in general, try to use the package name itself, to not confuse those using custom nicknames.